components/booking/HotelRooms/HotelRoomItem.tsx

```diff
@@ -41,67 +42,57 @@ const HotelRoomItem: React.FC<HotelRoomItemProps> = ({ room }) => {
   ).replace("{roomId}", room.hotel_room_id);
 
   return (
-    <div className="border rounded-lg p-4 shadow-md mb-4">
-      <h2 className="text-xl font-semibold mb-2 text-black">{room.name}</h2>
-
-      {/* Render the HTML description safely */}
-      <div
-        className="text-gray-700 mb-4"
-        dangerouslySetInnerHTML={{ __html: room.description }}
-      ></div>
-
-      {/* Price Per Night Section */}
-      {room.pricePerNight.amount > 0 ? (
-        <div className="text-lg font-medium text-black mb-4">
-          Price per night:{" "}
-          {`${room.pricePerNight.currency.symbol}
-          ${room.pricePerNight.amount} ${room.pricePerNight.currency.code}`}
-        </div>
-      ) : (
-        <div className="text-lg font-medium text-black mb-4">
-          Price per night: Unavailable
-        </div>
-      )}
-
-      {/* Room ID */}
-      <div className="text-sm text-gray-600 mb-4">
-        Room ID: {room.hotel_room_id}
+    <div className="flex flex-col gap-6 bg-base-200 rounded-box p-8 border border-primary shadow">
+      <div className="text-center text-base-content">
+        <h2 className="text-2xl font-semibold">{room.name}</h2>
+        <p className="text-lg text-base-content" dangerouslySetInnerHTML={{ __html: room.description }}>
+        </p>
       </div>
 
-      {/* Gallery Images */}
-      <div className="grid grid-cols-2 gap-2 mb-4">
-        {room.galleryImages.map((image) => (
-          <div key={image.index} className="flex flex-col items-center">
+      {/* Room Images */}
+      <div className="carousel carousel-center bg-neutral rounded-box space-x-4 h-96 p-4">
+      {room.galleryImages.map((image) => (
+          <div key={image.index} className="carousel-item">
             <Image
               src={image.url}
               alt={image.description}
               width={500}
               height={500}
               className="w-full h-auto rounded-lg"
             />
-            <p className="text-sm text-gray-500 mt-1">{image.description}</p>
           </div>
         ))}
       </div>
 
-      {/* Centered Custom Link and Reserve Now Button */}
+
+      {/* Pricing Section */}
+      {room.pricePerNight.amount > 0 ? (
+        <div className="text-center">
+          <p className="text-3xl font-bold text-primary">
+            {`${room.pricePerNight.currency.symbol}${(
+              room.pricePerNight.amount * numDays
+            ).toFixed(2)}`}
```

Comment:
I'd prefer to have calculations done before returning the html. This way it can be reused and we clearly separate ui form logic.